photos

```diff
@@ -0,0 +1,88 @@
+from fastapi import FastAPI, HTTPException, Depends
```

Comment:
In my own opinion I don't think the collaborators want third party applications because what if they get shut down! But that's up to you. I would say this is approved.